CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.0]
+### Changed
+- **BREAKING:** Update `Keyring` type ([#89](https://github.com/MetaMask/utils/pull/89))
+  - The `Keyring` class now uses the data types `TypedTransaction` and `TxData` from `@ethereumjs/tx` (`v4.1.1`).
+  - The `Keyring` now exposes a new optional method called `generateRandomMnemonic`.
+- Bump `peaceiris/actions-gh-pages` to more recent version ([#88](https://github.com/MetaMask/utils/pull/88))
```

Comment:
This is irrelevant for the public API, so does not need to be mentioned in the changelog.

Reply:
[2143316](https://github.com/MetaMask/utils/pull/93/commits/2143316e1dd6abec226fd0c7252b7621d91493b1)